### PR TITLE
Reusing profile

### PR DESCRIPTION
**Pull request by gustavoneves12**

### CODE REVIEW

**Comment by doudou** [comment]
OK. The main difference is on namespacing (it is documented in the main syskit documentation)

Namely, all actions (including Main) go in your case in Tutorials::Actions::, profiles in Tutorials::Profiles:: and so on.

File: models/actions/tut-transformer/main.rb [hunk 1]
```diff
@@ -0,0 +1,8 @@
+require 'models/profiles/rocks'
+module Tutorials
+    module Actions
+        class Main < Roby::Actions::Interface
```
Comment (doudou) [inline models/actions/tut-transformer/main.rb:4]: You don't need this anymore. This is basically done with the Robot.actions block in the robot's config file now.

File: models/compositions/rock_control.rb [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+
+using_task_library 'controldev'
+using_task_library 'tut_follower'
+using_task_library 'tut_sensor'
+using_task_library 'rock_tutorial'
+
+module Tutorials
+    class RockControl < Syskit::Composition
+
+        add Controldev::JoystickTask, as: 'joystick'
```
Comment (doudou) [inline models/compositions/rock_control.rb:13]: The models generated from the orogen components are now in the OroGen:: namespace. You can set `Roby.app.backward_compatible_naming = false` in the bundle's config/init.rb to disable the backward-compatible naming.

**Comment by gustavoneves12** [comment]
Now the changes will be made in the: https://github.com/gustavoneves12/bundle-tutorials/tree/bundle_file_structure

**Pull request merged**